src/client/rewards/MatchBot/CreateRule/DeleteRuleModal/DeleteRuleModal.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import { Button, Modal } from 'antd';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import { injectIntl } from 'react-intl';
+import './DeleteRuleModal.less';
+
+const DeleteRuleModal = ({
+  deleteLoading,
+  handleModalVisibility,
+  intl,
+  isDeleteModal,
+  onOk,
+  sponsor,
+}) => {
+  const handleDeleteRule = () => handleModalVisibility();
+  return (
+    <Modal
+      title={intl.formatMessage({
+        id: 'matchBot_modal_confirmation_required',
+        defaultMessage: 'Confirmation required',
+      })}
+      wrapClassName="DeleteRuleModal"
+      onCancel={handleDeleteRule}
+      visible={isDeleteModal}
+      footer={null}
+    >
+      <div className="DeleteRuleModal__content">
+        {intl.formatMessage({
+          id: 'matchBot_do_you_want_delete_match_bot_rules',
+          defaultMessage: 'Do you want to delete match bot rules for',
+        })}
+        <Link to={`/@${sponsor}`}>{` @${sponsor}`}</Link>?
+      </div>
+      <div className="DeleteRuleModal__footer">
+        <div className="CreateRule__button">
+          <Button onClick={handleModalVisibility} disabled={deleteLoading}>
```

Comment:
'handleModalVisibility' assigned to 'handleDeleteRule' - use one of them

Reply:
++